src/controllers/qaController.js

```diff
@@ -1,21 +1,74 @@
 const db = require('../db');
 
+const loopAnswersToGetPhotos = (answers, object) => {
+  return Promise.all(answers.map(answer=>{
+    return db.query('SELECT * FROM photos WHERE answer_id = $1',[answer.id])
```

Comment:
What does $1 do? I'm not familiar with that syntax

Reply:
$1 is just a variable for the query, so for instance, db.query('select ... where id = $1 and id = $2', [a,b]), $1 has the value 'a' in the argument array, and $2 has the value of 'b' in the argument array

Reply:
Just got to the part about jsonpath variables LOL makes sense now